src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -227,125 +226,120 @@ library BeaconChainProofs {
      * @notice This function verifies the latestBlockHeader against the state root. the latestBlockHeader is 
      * a tracked in the beacon state.
      * @param beaconStateRoot is the beacon chain state root to be proven against.
-     * @param latestBlockHeaderProof is the provided merkle proof
-     * @param latestBlockHeaderRoot is hashtree root of the latest block header in the beacon state
+     * @param stateRootProof is the provided merkle proof
+     * @param latestBlockRoot is hashtree root of the latest block header in the beacon state
      */
-    function verifyStateRootAgainstLatestBlockHeaderRoot(
+    function verifyStateRootAgainstLatestBlockRoot(
         bytes32 beaconStateRoot,
-        bytes32 latestBlockHeaderRoot,
```

Comment:
maybe we should switch latestBlockRoot and beaconStateRoot in the param ordering?  I think everywhere else it is (root, leaf, proof) as input order

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/189/commits/26c02279a2e18f8b6d4f98f7cc53db029628d056 cood callout